### ISSUE TITLE
Accessibility audits of "atom" components

### ISSUE THREAD

**Issue by alex-hull** [uswds/uswds#6365]
Secondary round of audits for "atom" components. These component are smaller components. They do not usually have other components within them.

**Comment by ryparker** [comment]
I'm closing this tracker and continuing the second round of small-component audits in [#6925](https://github.com/uswds/uswds/issues/6925#issuecomment-5730233593).

That issue retains the component states and input methods to check, existing findings, open child tasks, and any device or assistive-technology combinations we can't yet test. These audits haven't been marked as passed.

Please flag any missing requirement in #6925.
